cmd/main.go

```diff
@@ -269,3 +272,32 @@ func updateHealthMetricsPeriodically(nsxClient *nsx.Client) {
 		}
 	}
 }
+
+func checkLicense(nsxClient *nsx.Client, interval int) {
+	err := nsxClient.ValidateLicense(true)
```

Comment:
The name `checkLicense` and `ValidateLicense` look the same, I think maybe we can just remove the function checkLicense and move all logic in main

Reply:
although the name is similar,  checkLicense includes the logic of setting interval, spawn a go routine,  it's better to hidden those detail from main function. maybe change it name to spawnLicenseMonitor